tests/model/test_model.py

```diff
@@ -371,6 +371,13 @@ def test_get_message_false_first_anchor(
         model.get_messages(num_after=0, num_before=0, anchor=0)
         assert model.update is False
 
+        # TEST `query_range` == no_of messages received
+        # len(messages_successful_response) = 3 so comparing this with
+        # num_after + num_before + 1 = 3, should set update to True.
+        model.update = False
```

Comment:
I wonder if `model.is_upto_date` or `model.is_updated` is a better name, than `model.update`. But, I guess that can come as a separate change if we'd like to change it.

Reply:
Definitely! I will add the change as a seperate for now :)

Reply:
@amanagr What do you think of renaming `self.update` here?